bare-metal/apple-silicon/troubleshooting/cant-connect-using-vnc.mdx

```diff
@@ -39,19 +82,21 @@ If you are experiencing this issue, you can try the following solutions:
 
 2. Run the following command to forcefully terminate Screen Sharing:
     ```
-    sudo killall screensharing
+    sudo killall screensharingd
     ```
 
 3. Run the following command to load the `screensharing.plist` file responsible for Screen Sharing: 
```

Comment:
I'm not sure 3 and 4 are necessary to restart screensharingd. It most probably is required only to start or stop it, but if it has been started (as it should be by our macOS installation) then there should be no need for it.

Reply:
requires testing I guess

Reply:
Indeed, it restarts automatically